reconcile/dynatrace_token_provider/integration.py

```diff
@@ -66,11 +66,15 @@ def reconcile(self, dry_run: bool, dependencies: Dependencies) -> None:
         with metrics.transactional_metrics(self.name):
             unhandled_exceptions = []
             for ocm_env_name, ocm_client in dependencies.ocm_client_by_env_name.items():
-                clusters = ocm_client.discover_clusters_by_labels(
-                    label_filter=subscription_label_filter().like(
-                        "key", dtp_label_key("%")
-                    ),
-                )
+                clusters: list[Cluster] = []
+                try:
+                    clusters = ocm_client.discover_clusters_by_labels(
+                        label_filter=subscription_label_filter().like(
+                            "key", dtp_label_key("%")
+                        ),
+                    )
+                except Exception as e:
+                    unhandled_exceptions.append(f"{ocm_env_name}: {e}")
```

Comment:
If we need full stacktrace of excpetions, can do error logging and aggregate them together using [ExceptionGroup](https://docs.python.org/3/library/exceptions.html#ExceptionGroup) and raise at the end.

Reply:
Good idea! Wll for now keep the current mechanism, but will add that when I start making changes for DTPv2